app/models/accounting/qb/full_fetcher.rb

```diff
@@ -37,6 +37,7 @@ def fetch_qb_data
         ::Accounting::QB::TransactionFetcher.new(division).fetch
         ::Accounting::QB::DepartmentFetcher.new(division).fetch
         ::Accounting::QB::VendorFetcher.new(division).fetch
+        Rails.logger.debug("Setting qb cnxn last_updated_at to #{started_fetch_at}")
```

Comment:
Should we make a method on the Connection class for updating the stamp and move this log statement in there? That way it's not duplicated and we don't accidentally leave it out if we add one of these somewhere else?